app/models/persistent_volume.rb

```diff
@@ -1,4 +1,15 @@
 class PersistentVolume < ContainerVolume
   acts_as_miq_taggable
+  serialize :capacity
```

Comment:
I agree with @alongoldboim .  Though I am not opposed to hstore in the long term, I'd prefer this route for the short-term.